data/json/itemgroups/Clothing_Gear/clothing.json

```diff
@@ -1907,6 +1907,32 @@
       { "item": "quiver_takedown_bow", "prob": 1 }
     ]
   },
+  {
+    "type": "item_group",
+    "id": "large_bags",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "backpack", "prob": 38 },
+      { "item": "backpack_hiking", "prob": 3 },
+      { "item": "backpack_tactical_large", "prob": 1 },
+      { "item": "bigback", "prob": 1 },
+      { "item": "travelpack", "prob": 5 },
+      { "item": "rucksack", "prob": 20 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "small_bags",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "petpack", "prob": 3 },
+      { "item": "purse", "prob": 40 },
+      { "item": "mbag", "prob": 20 },
+      { "item": "slingpack", "prob": 8 },
+      { "item": "backpack_leather", "prob": 8 },
```

Comment:
As this is only missing the carabiner + tucked-in-back slots of the regular backpack, I don't think it'd be considered a small bag
(idk why it doesn't have tucked-in-back slot 🤔)